trio/_core/_io_epoll.py

```diff
@@ -12,12 +14,16 @@
 
 assert not TYPE_CHECKING or sys.platform == "linux"
 
+if TYPE_CHECKING:
+    from typing_extensions import Literal, TypeAlias
+EventResult: TypeAlias = "list[tuple[int, int]]"
+
 
 @attr.s(slots=True, eq=False, frozen=True)
 class _EpollStatistics:
-    tasks_waiting_read = attr.ib()
-    tasks_waiting_write = attr.ib()
-    backend = attr.ib(default="epoll")
+    tasks_waiting_read: int = attr.ib()
+    tasks_waiting_write: int = attr.ib()
+    backend: Literal["epoll"] = attr.ib(default="epoll")
```

Comment:
I really feel like this should have `init=False`

Reply:
Actually did that originally, but didn’t want to do a functional change.